app/assets/javascripts/components/accountPanel/sidebar.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+import _ from 'lodash';
+import { connect } from 'react-redux';
+import { Link } from 'react-router';
+import { Button } from 'react-bootstrap';
+import { actions as appActions } from 'redux/modules/app.js';
+
+export default function MenuItem({ menu }) {
+  return (
+    <li className="uctext">
+      <Link {...menu.linkProps} key={menu.label} activeClassName="active">
+        {menu.label}
+      </Link>
+    </li>
+  );
+}
+
+class Sidebar extends Component {
+  constructor(props) {
+    super(props);
+    console.log(props)
+  }
+
+  closeAccount() {
+    const { dispatch, tabId } = this.props;
+  }
+
+  renderMenuItems() {
+  const basePath = '/dashboard/account';
+  const menuItems = [
+    {
+      label: 'Details',
+      linkProps: {
+        to: basePath,
+        onlyActiveOnIndex: true,
+      },
+    },
+    {
+      label: 'Current Plan',
+      linkProps: {
+        to: `${basePath}/current_plan`,
+        onlyActiveOnIndex: false,
+      },
+    },
+  ];
+
```

Comment:
<img width="584" alt="screen shot 2016-10-10 at 9 00 21 pm" src="https://cloud.githubusercontent.com/assets/17052014/19255522/d9278c2e-8f2c-11e6-9baa-fdb55a4efa3c.png">

Can't figure out why the activeClassName is not working here..